src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureManagementConstants.cs

```diff
@@ -5,9 +5,28 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManage
 {
     internal class FeatureManagementConstants
     {
+        // Common
+        public const string FeatureManagementSchemaEnvironmentVariable = "AZURE_APP_CONFIGURATION_FEATURE_MANAGEMENT_SCHEMA_VERSION";
+        public const string FeatureManagementSchemaV1 = "1";
+        public const string FeatureManagementSchemaV2 = "2";
```

Comment:
We can introduce a `FeatureManagementDefaultSchemaVersion` so we don't have to go back and find out what code we need to change for the default next time.